controllers/tasks.js

```diff
@@ -52,8 +52,13 @@ const fetchTasks = async (req, res) => {
  */
 const getUserTasks = async (req, res) => {
   try {
+    const statuses = req.query.status
     const { username } = req.params
-    const allTasks = await tasks.fetchUserTasks(username)
+    let allTasks = []
+
+    if (statuses) {
+      allTasks = await tasks.fetchUserTasks(username, statuses.split(','))
```

Comment:
why are we splitting? we are not going to have the request that will look something like this - `tasks/:username?status=active,inactive`
this is invalid.
so you can do this

```javascript
await tasks.fetchUserTasks(username, [status])
```

Also rename `statuses` to `status`